css/properties/rotate.json

```diff
@@ -7,8 +7,7 @@
           "spec_url": "https://drafts.csswg.org/css-transforms-2/#individual-transforms",
           "support": {
             "chrome": {
-              "version_added": false,
-              "notes": "See <a href='https://crbug.com/496550'>bug 496550</a>."
+              "version_added": "104"
```

Comment:
Can you also check the "x_y_z_angle" in this file? What does it mean, and is it supported?